include/wx/private/webviewutils.h

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/private/webviewutils.h
+// Purpose:     Common code for wxWebView
+// Author:      Jose Lorenzo
+// Created:     2017-08-12
+// Copyright:   (c) 2017 Jose Lorenzo <josee.loren@gmail.com>
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_PRIVATE_WEBVIEWUTILS_H_
+#define _WX_PRIVATE_WEBVIEWUTILS_H_
+
+#include "wx/regex.h"
+
+// ----------------------------------------------------------------------------
+// Common wxWebView code
+// ----------------------------------------------------------------------------
+
+class wxWebViewUtils
+{
+public:
+
+    // This method is used to add a double quote level into a JavasSript code
+    // in order to get it working when eval is called programmatically.
+    static wxString WrapJavaScript
+        (const wxString& javascript, int* runScriptCount, wxString* counter)
```

Comment:
So, what I should do, refactor to src/common/webview.cpp or rename and refactor the class you mentioned?